src/App.js

```diff
@@ -36,19 +40,22 @@ function App() {
     <>
       {showLogin && <Login />}
       {showSignup && <Signup />}
+      {showSignupAlert && <SignupAlertModal />}
+      {showError && <AlertModal />}
       {editModal && <EditNoteModal />}
-      {hideNav && <NavBar />}
-      {hideNav && <MobileNavBar />}
+      {hideNav && auth.login && <NavBar />}
+      {hideNav && auth.login && <MobileNavBar />}
       <BodyWrapper>
-        {hideNav && <LeftNavBar />}
+        {hideNav && auth.login && <LeftNavBar />}
         <Routes>
           <Route path="/" element={<LandingPage />} />
-          <Route path="/home" element={<HomePage />} />
-          <Route path="/label" element={<LabelPage />} />
-          <Route path="/archive" element={<ArchivePage />} />
-          <Route path="/trash" element={<TrashPage />} />
-          <Route path="/profile" element={<ProfilePage />} />
+          {auth.login && <Route path="/home" element={<HomePage />} />}
+          {auth.login && <Route path="/label" element={<LabelPage />} />}
```

Comment:
Use a ternary and replace them to login page

Reply:
Noted